### ISSUE TITLE
When removing the context parent in the context popup also show this on the parent

### ISSUE THREAD

**Issue by creesch** [toolbox-team/reddit-moderator-toolbox#345]
The title is a bit confusing but basically if you click the context-popup for a comment and end up removing that comment from the popup I'd like to also show that comment as removed when closing the popup. This is probably only easily possible on old reddit but that is fine by me. 

As we also have other functionality in the queuetools tied to removals it might be easiest to actually do the removal through clicking the native remove button. 

As a bit of background, recently I have enabled the setting that hides things I have removed in the queue. But when using the context popup and hitting remove there the thing remains visible in the queue without any indication I already have taken action on it. 

I also realize this might not be easily implemented without making it hacky, so practical implementation needs to be looked at first.

**Comment by Geo1088** [comment]
I think snoonotes already provides this functionality indirectly by showing removals from other mods in the queue. Haven't looked into the actual implementation but it seems likely that it fetches modlog periodically and updates classes and adds action info for posts that have been auctioned since page load.

I don't know specifically if it works for comments you remove yourself from another place, but in theory we should be able to take some ideas from that implementation.

**Comment by creesch** [comment]
Hrm that makes it a lot more complex though, could be a nice feature besides this. 

I literally am talking about things happening on the same page: 

1. Comment in queue
2. Click context popup
3. Click remove on that exact same comment in the popup.
4. Close popup 

After step 4 I'd like the comment still visible in the queue to be marked as removed in the reddit native way without delay.

**Comment by Geo1088** [comment]
Yeah, I got that, just pointing out some prior art in the same vein. I don't want our implementation to be that complicated, if anything we should just make sure whatever we end up doing doesn't conflict with what snoonotes does.
